recipes/aiida-core/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set name = "aiida-core" %}
+{% set version = "0.12.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 482e7aff408d6cc21e8ddde22efaa124d71619de53eccd37174c8e6d7e7e141e
+
+build:
+  number: 0
+  entry_points:
+    - verdi=aiida.cmdline.verdilib:run
```

Comment:
if possible you might consider testing these

Reply:
I originally tested the `verdi` command and it does seem to work as intended but, due to an oversight, both `verdi` and `verdi help` (which would be easy to  test) return exit status 1. See [travis log]https://travis-ci.org/conda-forge/staged-recipes/builds/466539864#L2263.

I've [opened an issue for this](https://github.com/aiidateam/aiida_core/issues/2355) on the aiida side - would it be fine to include the test with the 0.12.3 release?

The `verdi-plug` entry point was outdated and has been removed.